src/app/shared/top-panel/user-panel/notifications-menu/notifications-menu.component.scss

```diff
@@ -0,0 +1,143 @@
+@import '../../../../../styles/variables';
+
+.panel-item.notifications {
+  height: var(--top-panel-line-height);
+  line-height: var(--top-panel-line-height);
+  white-space: nowrap;
+
+  .dropdown-menu {
+    width: 18rem;
+    max-width: 90vw;
+
+    form {
+      line-height: 1.1rem;
+    }
+  }
+
+  span.notifications-count {
+    font-size: 0.6rem;
+    padding: 0 3px;
+    color: white;
+    background-color: red;
+    border-radius: 10px;
+    position: absolute;
+    top: 0px;
+    right: 0px;
+    line-height: 0.9rem;
+    min-width: 0.9rem;
+    text-align: center;
+  }
+
+  a.dropdown-item {
+    line-height: initial;
+
+    &:active,
+    &:focus {
+      .notification-content {
+        &.unread {
+          color: $gray-500;
+        }
+      }
+    }
+
+    &:not(:last-child) {
+      padding-bottom: 1rem;
+    }
+
+    .notification-title {
+      display: inline-block;
+      width: 100%;
+      line-height: 1.2rem;
+      vertical-align: top;
+    }
+
+    .status {
+      vertical-align: baseline;
+    }
+
+    .notification-content {
+      display: block;
+      width: 100%;
+      line-height: 1.15rem;
+      white-space: normal;
+      text-align: justify;
+
+      &.unread {
+        color: $primary;
+        font-weight: bold;
+        &:active,
+        &:focus {
+          color: $light;
+        }
+      }
+
+      &.read,
+      &.read:hover,
+      &.read:focus {
+        font-weight: lighter;
+        font-style: italic;
+        color: $gray-500;
+      }
+    }
+  }
+
+  .switch {
+    position: relative;
+    display: inline-block;
+    width: 2.1rem;
+    height: 1.1rem;
+
+    input {
+      opacity: 0;
+      width: 0;
+      height: 0;
+    }
+
+    .slider {
+      position: absolute;
+      cursor: pointer;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      background-color: #ccc;
+      -webkit-transition: 0.4s;
+      transition: 0.4s;
+
+      &:before {
+        position: absolute;
+        content: '';
+        height: 1rem;
+        width: 1rem;
+        left: 0.05rem;
+        bottom: 0.05rem;
+        background-color: white;
+        -webkit-transition: 0.4s;
+        transition: 0.4s;
+      }
+
+      /* Rounded sliders */
+      &.round {
+        border-radius: 1rem;
+
+        &:before {
+          border-radius: 50%;
+        }
+      }
+    }
+
+    input:checked + .slider {
+      background-color: $success;
+    }
+
+    input:checked + .slider:before {
+      -webkit-transform: translateX(0.95rem);
+      -ms-transform: translateX(0.95rem);
+      transform: translateX(0.95rem);
```

Comment:
You do not have to add this browser-specific styles as they are added automatically by Angular CLI (using [Autoprefixer](https://github.com/postcss/autoprefixer)).